DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: ospsuite
 Title: R package to manipulate OSPSuite Models
-Version: 11.0.0
+Version: 12.0.0
```

Comment:
Now we should set the version to 12.0.0.**9000** , right? In order to avoid the production docs are overwritten by dev docs

Reply:
Sure. Done so in https://github.com/Open-Systems-Pharmacology/OSPSuite-R/pull/977.

Reply:
Are nightly builds failing because we bumped the version to `12.x.x` here and AppVeyor is expecting `11.x.x`?